presto-verifier/src/main/java/com/facebook/presto/verifier/event/QueryInfo.java

```diff
@@ -48,6 +48,7 @@
     private final List<String> teardownQueries;
     private final String checksumQuery;
     private final String jsonPlan;
+    private final String temporaryTableName;
```

Comment:
Let's call it `outputTableName`. Here and below. You can use the refactor functionality in IntelliJ